src/Pages/ProductDetail/Components/AddReview/AddReview.js

```diff
@@ -0,0 +1,165 @@
+import React, { Component } from 'react';
+import './AddReview.scss';
+
+export default class AddReview extends Component {
+  constructor() {
+    super();
+    this.state = {
+      fileArr: [],
+      review: [],
+      reviewContents: '',
+      star: '5',
+    };
+  }
+
+  handleFileInput = e => {
+    const { fileArr } = this.state;
+    const { files } = e.target;
+    this.setState(
+      {
+        fileArr: [
+          ...fileArr,
+          {
+            id: fileArr.length + 1,
+            src: window.URL.createObjectURL(files[0]),
+          },
+        ],
+      },
+      () => {
+        // console.log(fileArr);
+      }
+    );
+  };
+
+  handleReviewValue = e => {
+    const { value, name } = e.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  deletePhoto = id => {
+    const { fileArr } = this.state;
+    this.setState({
+      fileArr: fileArr.filter(file => file.id !== id),
+    });
+  };
+
+  addReview = () => {
+    const { reviewContents, review, star } = this.state;
+    if (!reviewContents) {
+      alert('리뷰 내용을 입력해주세요.');
+    } else {
+      this.setState(
+        {
+          review: [
+            ...review,
+            {
+              star,
+              reviewContents,
+            },
+          ],
+        },
```

Comment:
```suggestion
      this.setState({ review: [ ...review, { star, reviewContents }]},
```
줄바꿈이 너무 많이 되어 있는데 가독성을 위해 한줄로 표기할 수 있는 부분은 줄여주세요!

Reply:
네 알겠습니다!